jiant/scripts/postproc/README.md

```diff
@@ -0,0 +1,15 @@
+# NLP Benchmark Submission Formatter Scripts
+
+## GLUE Benchmark Submission Formatter
+
+Jiant supports [GLUE Benchmark](https://gluebenchmark.com/) test prediction generation in the submission format. To generate test predictions, use the `--write_test_preds` flag in [runscript.py](https://github.com/jiant-dev/jiant/blob/master/jiant/proj/main/runscript.py) when running your workflow. This will generate a `test_preds.p` file in the specified output directory. To convert `test_preds.p` to the required GLUE submission format, run the following:
```

Comment:
nit, `jiant`, because we're maybe stuck with this stupid convention.

Reply:
"test prediction generation in the submission format" => "generating submission files".